src/ReinforcementLearningZoo/src/algorithms/offline_rl/NFQ.jl

```diff
@@ -0,0 +1,78 @@
+"""
+    NFQ{A<:AbstractApproximator, F, R} <: AbstractLearner
+    NFQ(approximator::A, num_iterations::Integer epochs::Integer, loss_function::F, batch_size::Integer, rng::R, γ::Float32) where {A<:AbstractApproximator, F, R}
+Neural Fitted Q-iteration as implemented in [1]
+
+# Keyword arguments
+- `approximator::AbstractApproximator` neural network
```

Comment:
```suggestion
- `approximator:: NeuralNetworkApproximator` neural network
```

Reply:
I think `NeuralNetworkApproximator` does not exist anymore. It's only `Approximator` now as it can totally be a linear approximator as well.